### ISSUE TITLE
No PlotWidget has been specified.

### ISSUE THREAD

**Issue by mohammadsamani** [toolsforexperiments/plottr#235]
When I install the plottr, it runs, loads my database, shows the rows, but does not plot my data. It gives me the following error when I double click on a row.
Python version is 3.8. Everything was installed fresh yesterday on this machine.

```PS C:\Users\WindowsUser> plottr-inspectr
2021-11-29 16:28:31 - plottr.apps.inspectr - INFO
    Opening: C:/Users/WindowsUser/newdatabase.db
Traceback (most recent call last):
  File "C:\Users\WindowsUser\anaconda3\envs\qcodes\lib\site-packages\plottr\apps\inspectr.py", line 590, in plotRun
    fc, win = autoplotQcodesDataset(pathAndId=(self.filepath, runId))
  File "C:\Users\WindowsUser\anaconda3\envs\qcodes\lib\site-packages\plottr\apps\autoplot.py", line 332, in autoplotQcodesDataset
    win = QCAutoPlotMainWindow(fc, pathAndId=pathAndId,
  File "C:\Users\WindowsUser\anaconda3\envs\qcodes\lib\site-packages\plottr\apps\autoplot.py", line 277, in __init__
    super().__init__(fc, parent, **kw)
  File "C:\Users\WindowsUser\anaconda3\envs\qcodes\lib\site-packages\plottr\apps\autoplot.py", line 139, in __init__
    super().__init__(parent, fc=fc, plotWidgetClass=plotWidgetClass,
  File "C:\Users\WindowsUser\anaconda3\envs\qcodes\lib\site-packages\plottr\gui\widgets.py", line 100, in __init__
    raise RuntimeError("No PlotWidget has been specified.")
RuntimeError: No PlotWidget has been specified.```

**Comment by mohammadsamani** [comment]
An empty ```__init__.py``` file exists in this folder. The problem persists.
```C:\Users\WindowsUser\anaconda3\envs\qcodes\Lib\site-packages\plottr\config\```

**Comment by jenshnielsen** [comment]
Did you try the latest release? We have released 0.8.1 with a fix for this issue. 
Adding an empty init file to that folder will not in it self solve the issue. The problem is that since the init file was not there when the 0.8.0 package was generated none of the other files in that folder were included in the package. Specifically `plottrcfg_main.py` is missing. That file should be in that folder too.

**Comment by mohammadsamani** [comment]
Installing version 0.8.1 fixed the problem. Thank you!
